packages/headless/src/features/facets/facet-set/interfaces/options.ts

```diff
@@ -1,6 +1,8 @@
-import {FacetRequest} from './request';
+import {FacetOptions} from '../../../../controllers';
```

Comment:
we should avoid importing from `index.ts` files since this may mess with tree-shaking. you should instead import this from the exact file that defines it.